src/Model/ValueObject/Taxonomy/Category/CategoryDomain.php

```diff
@@ -20,4 +20,9 @@ public function __construct(string $value)
         $this->guardNotEmpty($value);
         $this->setValue($value);
     }
+
+    public static function eventType(): CategoryDomain
+    {
+        return new CategoryDomain('eventtype');
```

Comment:
You could use a constant for this

Reply:
But where would we reuse that constant?

Reply:
For me that does not matter, I just try to avoid strings in my code anywhere (except tests).
Currently it is only used once, but code gets changed, and in the future somebody might need that value again, promoting code reuse if the const already is there.

Reply:
I can turn the question around. Why would this string literal ever be exposed, it is contained in this class?

If you need a `CategoryDomain` of the type `eventtype` you can use `CategoryDomain::eventType()` and if you need the string literal you can use `toString`. So I would say by adding the `const` and making it `public` you break the encapsulation and open it for usage as a `string` again instead of a typed class.

If you could start from scratch this would have been a simple enum, but because of the legacy data we need to take into account unexpected values.